source/extensions/filters/listener/proxy_protocol/proxy_protocol.cc

```diff
@@ -48,10 +50,60 @@ namespace Extensions {
 namespace ListenerFilters {
 namespace ProxyProtocol {
 
+constexpr absl::string_view kProxyProtoStatsPrefix = "downstream_proxy_proto.";
```

Comment:
Arguably the upstream and downstream won't get confused because they have completely different metrics (very little to emit when producing a proxy proto header) and users can separate them with a `stat_prefix` (which I'm hoping you'll add in a followup PR), so this could be shortened to `proxy_proto`. But if you strongly prefer to include `downstream` in the name, I'm ok with that.

Reply:
Sure, I don't feel strongly. Changed to `proxy_proto`. Will follow-up with stat prefix PR.